components/automate-chef-io/content/docs/quickstart.md

```diff
@@ -62,37 +104,16 @@ Now that you have the `Vagrantfile` in your current directory, provision a new V
 vagrant up
 ```
 
-If this is your first time provisioning a VM, this operation may take a few
-minutes to download the base OS image.
+If this is your first time provisioning a VM, this operation may take a few minutes to download the base OS image.
 
-## Access the VM and Install Chef Automate
-
-1. Open a terminal session on your VM:
-
-     ```bash
-     vagrant ssh
-     ```
-
-1. Download and unzip the command line tool:
-
-     ```bash
-     curl https://packages.chef.io/files/current/automate/latest/chef-automate_linux_amd64.zip | gunzip - > chef-automate && chmod +x chef-automate
-     ```
-
-1. Run
-
-     ```bash
-     sudo ./chef-automate deploy
-     ```
+When the command completes, you can access the web UI by browsing to `https://chef-automate.test/` (assuming you followed the above instructions and created an entry in `/etc/hosts`).
```

Comment:
I would remove the parenthetical expression.

Reply:
Good point.